src/components/ModalWithFormSubmit.js

```diff
@@ -0,0 +1,32 @@
+import Modal from "./Modal.js";
+
+export default class ModalWithFormSubmit extends Modal {
+  _setSubmitAction(submitCallback) {
+    this._submitCallback = submitCallback;
+    this._submitButton = document.querySelector("#modal__confirm-card-delete");
```

Comment:
`    this._submitButton = document.querySelector("#modal__confirm-card-delete");
`

Please, find the button in line 11 only once